doc/arkode/guide/source/Introduction.rst

```diff
@@ -123,23 +123,30 @@ Changes in v5.4.0
 
 CMake 3.18.0 or newer is now required for CUDA support.
 
-A C++14 compliant compiler is now required for C++ based features and examples e.g., CUDA, HIP, RAJA, Trilinos, SuperLU_DIST, MAGMA, and GINKGO.
+A C++14 compliant compiler is now required for C++ based features and examples
+e.g., CUDA, HIP, RAJA, Trilinos, SuperLU_DIST, MAGMA, GINKGO, and KOKKOS.
 
 Added support for GPU enabled SuperLU_DIST and SuperLU_DIST v8.x.x. Removed
 support for SuperLU_DIST v6.x.x or older. Fix mismatched definition and
 declaration bug in SuperLU_DIST matrix constructor.
 
+Added support for the `Ginkgo <https://ginkgo-project.github.io/>`_  linear
+algebra library. This support includes new ``SUNMatrix`` and ``SUNLinearSolver``
+implementations, see the sections :numref:`SUNMatrix.Ginkgo` and
+:numref:`SUNLinSol.Ginkgo`.
+
+Added new ``NVector``, ``SUNMatrix``, and ``SUNLinearSolver`` implementations
+for interfacing with the `Kokkos Ecosystem <https://kokkos.org/>`_ for
+performance portability, see sections :numref:`NVectors.Kokkos`,
+:numref:`SUNMatrix.Kokkos`, and :numref:`SUNLinSol.Kokkos` for more information.
```

Comment:
Should this line in the user guides specify that matrix and linear solver support in Kokkos is currently relegated to dense matrices?  In the main `CHANGELOG.md` we instead specify these references as `SUNMATRIX_KOKKOSDENSE` and
`SUNLINEARSOLVER_KOKKOSDENSE`, but I anticipate that most users will read these and not that file.